model/installation.go

```diff
@@ -44,6 +44,11 @@ type Installation struct {
 	configMergeGroupSequence int64
 }
 
+// InstallationsCount represents the number of installations
+type InstallationsCount struct {
+	Count int `json:"installations_count"`
```

Comment:
I would remove the JSON tag here and simply return `Count` to stay similar to the other model structs.